doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,840 @@
+
+      * ``flash_and_run``
+      * ``heap_memory``
+      * ``static_memory``
+      * ``multiple_threads``
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+|no_changes_yet_note|
+
+Wi-Fi drivers
+-------------
+
+* Added the :ref:`nrf70_wifi_tx_power_calculation` section to the :ref:`nrf70_wifi` page.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Updated the :ref:`req_appln_limitations` page to clarify carrier-specific requirements.
+    Added overlay files and documentation to :ref:`serial_lte_modem` application and :ref:`lwm2m_carrier` sample to guide in the correct usage of LwM2M carrier library for SoftBank and LG U+.
+
+Bluetooth libraries and services
+--------------------------------
+
+
+* Added the :ref:`rreq_readme` and :ref:`rrsp_readme` libraries.
+
+* :ref:`hogp_readme` library:
+
+  * Updated the :c:func:`bt_hogp_rep_read` function to forward the GATT read error code through the registered user callback.
+    This ensures that API user is aware of the error.
+
+Common Application Framework
+----------------------------
+
+|no_changes_yet_note|
+
+Debug libraries
+---------------
+
+|no_changes_yet_note|
+
+DFU libraries
+-------------
+
+|no_changes_yet_note|
+
+Gazell libraries
+----------------
+
+|no_changes_yet_note|
+
+Security libraries
+------------------
+
+|no_changes_yet_note|
+
+Modem libraries
+---------------
+
+* :ref:`modem_key_mgmt` library:
+
+  * Updated to handle generic CME errors from all ``AT%CMNG`` commands.
+
+Multiprotocol Service Layer libraries
+-------------------------------------
+
+|no_changes_yet_note|
+
+Libraries for networking
+------------------------
+
+|no_changes_yet_note|
+
+Libraries for NFC
+-----------------
+
+|no_changes_yet_note|
+
+nRF RPC libraries
+-----------------
+
+* Added the :ref:`nrf_rpc_dev_info` library for obtaining information about a device connected through the :ref:`nrfxlib:nrf_rpc`.
+
+Other libraries
+---------------
+
+|no_changes_yet_note|
+
+Security libraries
+------------------
+
+|no_changes_yet_note|
+
+Shell libraries
+---------------
+
+|no_changes_yet_note|
+
+Libraries for Zigbee
+--------------------
+
+|no_changes_yet_note|
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`nrf_desktop_config_channel_script` Python script:
+
+  * Added support for pure ED25519 signature (used by nRF54L based devices that enable MCUboot hardware cryptography).
+    This requires using ``imgtool`` supporting pure ED25519 signature that can be installed from ``sdk-mcuboot`` repository.
+    See the script documentation page for details.
```

Comment:
```suggestion
    See the :ref:`script documentation page <nrf_desktop_config_channel_script>` for details.
```

Reply:
Not necessary to add the link as it is mentioned already above.